tests/requirements-robot.txt

```diff
@@ -0,0 +1,2 @@
+robotframework-requests==0.5.0

```

Comment:
I think these deps should be added to the rest of dev dependencies. Is there a reason why they should be kept separate?

Reply:
I thought that if unit tests were to fail it would shorten execution time, by nit installing RFWK libs, but it's beign run nevertheles, so I will move them to dev deps.